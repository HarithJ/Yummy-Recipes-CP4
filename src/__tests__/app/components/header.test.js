import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson, { shallowToJson } from 'enzyme-to-json';

import Header from '../../../components/auth/header.js'

describe("header component", () => {
  const header = shallow(<Header />)

  it("renders properly", () => {
    expect(shallowToJson(header)).toMatchSnapshot();
  })
})
