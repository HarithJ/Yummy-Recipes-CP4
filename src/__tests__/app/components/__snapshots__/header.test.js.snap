// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component renders properly 1`] = `
<div
  className="row header"
>
  <div
    className="col-4 align-self-center header-text"
  >
    <h1
      className="text-center"
    >
      Yummy Recipes
    </h1>
  </div>
  <div
    className="col-3 header-img1"
  >
    <img
      alt=""
      className="img-fluid img-thumbnail"
      src="recipe.jpg"
      style={
        Object {
          "transform": "rotate(20deg)",
        }
      }
    />
  </div>
  <div
    className="col-3 align-self-center header-img2"
  >
    <img
      alt=""
      className="img-fluid img-thumbnail text-right"
      src="food.jpg"
    />
  </div>
</div>
`;
